packages/flutter/test/material/tabs_utils.dart

```diff
@@ -0,0 +1,239 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/material.dart';
+import 'package:flutter/rendering.dart';
+import 'package:flutter_test/flutter_test.dart';
+
+RenderParagraph getTabText(WidgetTester tester, String text) {
+  return tester.renderObject<RenderParagraph>(find.text(text));
+}
+
+TabController getTabController({required int length, required TickerProvider vsync, int initialIndex = 0, Duration? animationDuration}) {
+  final TabController result = TabController(length: length, vsync: vsync, initialIndex: initialIndex, animationDuration: animationDuration);
```

Comment:
Nit, these are probably better read broken out over multiple lines
Also, createTabController sounds better IMO, since it is creating a new one. :)

```suggestion
TabController createTabController({
  required int length,
  required TickerProvider vsync,
  int initialIndex = 0,
  Duration? animationDuration,
}) {
  final TabController result = TabController(
    length: length,
    vsync: vsync,
    initialIndex: initialIndex,
    animationDuration: animationDuration,
  );
```